content/Mobile_Dev/WebAuthn/WebAuthn_Primer.adoc

```diff
@@ -0,0 +1,54 @@
+== WebAuthn primer for mobile development
+This section should be referenced if you are looking to begin development for a mobile application, but are unfamiliar with WebAuthn concepts and development. The goal of this page is threefold:
+
+* Point you in the direction of  resources to learn more about the WebAuthn protocol
+* Provide additional considerations for developing WebAuthn on mobile devices
+* Guide developers on how to deploy an example WebAuthn application.
+
+=== Where can I learn more about WebAuthn?
+Below are a variety of resources for you to continue to learn more about WebAuthn 
+
+* link:/WebAuthn/WebAuthn_Developer_Guide/[Yubico WebAuthn Developer Guide] - Overview of a variety of topics to help you become familiar with WebAuthn concepts
+* link:https://www.edx.org/course/introduction-to-web-authentication[Introduction to WebAuthn] - Course developed by W3C and Yubico meant to introduce developers to the WebAuthn API and concepts
+* link:https://www.w3.org/TR/webauthn-2/[WebAuthn specification] - W3C specification for the WebAuthn protocol.
+* link:https://github.com/YubicoLabs/WebAuthnKit[WebAuthn Starter Kit] - Open source project for you to create your own test environment
+
+=== Important concepts to consider
+The following subsections will outline considerations to take during the implementation of your application. While the concepts below aren’t inherently specific to mobile applications with WebAuthn, it’s worth reinforcing them to help you frame your implementation and UX strategy.
+
+==== Platform vs Cross Platform authenticators
+The WebAuthn standard supports two types of authenticators:
+
+Platform authenticators, also known as internal authenticators, such as a fingerprint scanner built into a smartphone. Platform authenticators are limited to authenticating a user via a specific device (in the case of Windows Hello, the laptop running it). To support platform authenticators, a device must include both:
```

Comment:
Consider using a table here to compare and contrast authenticatorattachment types

Reply:
I tried a few different variations of a table and they didn't sit right with me

I think before we commit to a full comparison we need to bring our heads together on messaging around how passkeys play into the platform authenticator space

We can discuss later if needed, but for now I opt'ed to leave this out